servicetalk-annotations/build.gradle

```diff
@@ -17,7 +17,5 @@
 apply plugin: "io.servicetalk.servicetalk-gradle-plugin-internal-library"
 
 dependencies {
-  implementation platform(project(":servicetalk-dependencies"))
-
-  implementation "com.google.code.findbugs:jsr305"
+  implementation "com.google.code.findbugs:jsr305:$jsr305Version"
```

Comment:
I know this is slightly offtopic, but would it make sense to expose this as api instead of implementation? then I think in i.e. buffer-api we do not need to repeat `implementation "com.google.code.findbugs:jsr305:$jsr305Version"` if the annotations are already imported (and we expose it as api, not an implementation detail)

Reply:
yes, you are right. I will do a follow-up for this suggestion

Reply:
👍

Reply:
See #3056